src/pages/Tasks/components/board/TaskBoard.tsx

```
@@ -0,0 +1,30 @@
+  const { tasks } = useTask();
+  const stages = Array.from(Object.values(TaskStatus), (status) => status);
+  const { task, setTask } = useTaskGlobal();
+  console.log("🚀 ~ TaskBoard ~ task:", task);
+  return (
+    <div className="flex gap-4 w-fit pr-4">
+      {stages.map((stage) => (
+        <Stage
+          tasks={tasks.filter((x) => x.status === stage)}
+          title={stageNames[stage]}
+        />
+      ))}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)